Common/src/geometry_structure.cpp

```diff
@@ -9515,167 +9666,1804 @@ void CPhysicalGeometry::SetVertex(CConfig *config) {
+            TurboRadius[iMarker]                      = new su2double [nSpanWiseSections[marker_flag-1] + 1];
+            for (iSpan= 0; iSpan < nSpanWiseSections[marker_flag-1] + 1; iSpan++){
+              AverageTurboNormal[iMarker][iSpan]      = new su2double [nDim];
+              AverageNormal[iMarker][iSpan]           = new su2double [nDim];
+              AverageGridVel[iMarker][iSpan]          = new su2double [nDim];
+            }
+          }
+          for (iSpan= 0; iSpan < nSpanWiseSections[marker_flag-1] + 1; iSpan++){
+            AverageTangGridVel[iMarker][iSpan]          = 0.0;
+            SpanArea[iMarker][iSpan]                    = 0.0;
+            TurboRadius[iMarker][iSpan]                 = 0.0;
+            for(iDim=0; iDim < nDim; iDim++){
+              AverageTurboNormal[iMarker][iSpan][iDim]  = 0.0;
+              AverageNormal[iMarker][iSpan][iDim]       = 0.0;
+              AverageGridVel[iMarker][iSpan][iDim]      = 0.0;
+            }
+          }
+        }
+      }
+    }
+  }
+
+
+
+  /*--- start computing the average quantities span wise --- */
+  for (iSpan= 0; iSpan < nSpanWiseSections[marker_flag-1]; iSpan++){
+
+    /*--- Forces initialization for contenitors to zero ---*/
+    for (iDim=0; iDim<nDim; iDim++) {
+      TotalTurboNormal[iDim]	=0.0;
+      TotalNormal[iDim]         =0.0;
+      TotalGridVel[iDim]        =0.0;
+    }
+    TotalArea = 0.0;
+    TotalRadius = 0.0;
+    for (iMarker = 0; iMarker < config->GetnMarker_All(); iMarker++){
+      for (iMarkerTP=1; iMarkerTP < config->GetnMarker_Turbomachinery()+1; iMarkerTP++){
+        if (config->GetMarker_All_Turbomachinery(iMarker) == iMarkerTP){
+          if (config->GetMarker_All_TurbomachineryFlag(iMarker) == marker_flag){
+            for(iVertex = 0; iVertex < nVertexSpan[iMarker][iSpan]; iVertex++){
+              iPoint = turbovertex[iMarker][iSpan][iVertex]->GetNode();
+              turbovertex[iMarker][iSpan][iVertex]->GetTurboNormal(TurboNormal);
+              turbovertex[iMarker][iSpan][iVertex]->GetNormal(Normal);
+              coord  = node[iPoint]->GetCoord();
+
+              if (nDim == 3){
+                radius = sqrt(coord[0]*coord[0] + coord[1]*coord[1]);
+              }
+              else{
+                radius = 0.0;
+              }
+              Area = turbovertex[iMarker][iSpan][iVertex]->GetArea();
+              TotalArea   += Area;
+              TotalRadius += radius;
+              for (iDim = 0; iDim < nDim; iDim++) {
+                TotalTurboNormal[iDim]  +=TurboNormal[iDim];
+                TotalNormal[iDim]       +=Normal[iDim];
+              }
+              if (grid_movement){
+                gridVel = node[iPoint]->GetGridVel();
+                for (iDim = 0; iDim < nDim; iDim++) TotalGridVel[iDim] +=gridVel[iDim];
+              }
+            }
+          }
+        }
+      }
+    }
+
+#ifdef HAVE_MPI
+
+    MyTotalArea 
```

Comment:
Is there any change in CPhysicalGeometry::SetCoord_CG? It looks like that the diff tool is not working properly